package.json

```diff
@@ -56,14 +56,15 @@
     "cookie-parser": "~1.4.4",
     "cors": "^2.8.5",
     "debug": "~2.6.9",
+    "docker": "^1.0.0",
```

Comment:
Do we need the docker dependency here? I would prefer maintaining the package.json we have already, but if this is necessary, we can keep it.

Reply:
As far as I know I didn't touch the package.json file so I don't really know how that got in there.